src/resolvers/userResolver.js

```diff
@@ -134,6 +134,47 @@ const userResolver = {
       );
     },
 
+    changeUsername: async(_, { userInfo }, { req }) => {
+      const { newUsername } = userInfo;
+      const oldUser = await User.findById(req.session.userId);
```

Comment:
We are not using the const `oldUser` so we can get rid it. We can combine 139 and 140 into one line.